### PR TITLE
useradd, userdel, usedmod, users: refresh page

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [ ] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

**Comment by EmilySeville7cfg** [comment]
It is supposed that when `_name` suffix is not used it is applied implicitly. For instance when I write `user` instead of `user_name` I assume `user_name` by default.

File: pages/common/users.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 # users
 
 > Display a list of logged in users.
-> More information: <https://www.gnu.org/software/coreutils/users>.
+> More information: <https://manned.org/man/users.1>.
 
-- Display a list of logged in users:
+- Print logged in user names:
```
Comment (navarroaxel) [inline pages/common/users.md:6]:
```suggestion
- Print logged in usernames:
```

File: pages/common/users.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 
 `users`
 
-- Display a list of logged in users according to a specific file:
+- Print logged in user names according to a given file:
```
Comment (navarroaxel) [inline pages/common/users.md:10]:
```suggestion
- Print logged in usernames according to a given file:
```

File: pages/linux/useradd.md [hunk 3]
```diff
@@ -3,22 +3,30 @@
 > Create a new user.
 > More information: <https://manned.org/useradd>.
 
-- Create new user:
+- Create a new user:
 
-`useradd {{name}}`
+`useradd {{user}}`
```
Comment (navarroaxel) [inline pages/linux/useradd.md:8]:
```suggestion
`useradd {{username}}`
```

File: pages/linux/useradd.md [hunk 4]
```diff
@@ -3,22 +3,30 @@
...
 
-- Create new user with a default home directory:
+- Create a new user with the specified user id:
 
-`useradd --create-home {{name}}`
+`useradd --uid {{id}} {{user}}`
```
Comment (navarroaxel) [inline pages/linux/useradd.md:12]:
```suggestion
`useradd --uid {{id}} {{username}}`
```

File: pages/linux/useradd.md [hunk 5]
```diff
@@ -3,22 +3,30 @@
...
 
-- Create new user with specified shell:
+- Create a new user with the specified shell:
 
-`useradd --shell {{path/to/shell}} {{name}}`
+`useradd --shell {{path/to/shell}} {{user}}`
```
Comment (navarroaxel) [inline pages/linux/useradd.md:16]:
```suggestion
`useradd --shell {{path/to/shell}} {{username}}`
```

File: pages/linux/useradd.md [hunk 6]
```diff
@@ -3,22 +3,30 @@
...
 
-- Create new user belonging to additional groups (mind the lack of whitespace):
+- Create a new user belonging to additional groups (mind the lack of whitespace):
 
-`useradd --groups {{group1,group2}} {{name}}`
+`useradd --groups {{group_a,group_b}} {{user}}`
```
Comment (navarroaxel) [inline pages/linux/useradd.md:20]: We use `1, 2` for sequences
```suggestion
`useradd --groups {{group1,group2}} {{username}}`
```

File: pages/linux/useradd.md [hunk 7]
```diff
@@ -3,22 +3,30 @@
...
 
-- Create new system user without a home directory:
+- Create a new user with the default home directory:
 
-`useradd --no-create-home --system {{name}}`
+`useradd --create-home {{user}}`
+
+- Create a new user with the home directory filled by template directory files:
+
+`useradd --skel {{path/to/template_directory}} --create-home {{user}}`
```
Comment (navarroaxel) [inline pages/linux/useradd.md:28]:
```suggestion
`useradd --skel {{path/to/template_directory}} --create-home {{username}}`
```

File: pages/linux/useradd.md [hunk 8]
```diff
@@ -3,22 +3,30 @@
...
+- Create a new user belonging to additional groups (mind the lack of whitespace):
 
-`useradd --groups {{group1,group2}} {{name}}`
+`useradd --groups {{group_a,group_b}} {{user}}`
 
-- Create new system user without a home directory:
+- Create a new user with the default home directory:
 
-`useradd --no-create-home --system {{name}}`
+`useradd --create-home {{user}}`
```
Comment (navarroaxel) [inline pages/linux/useradd.md:24]:
```suggestion
`useradd --create-home {{username}}`
```

File: pages/linux/useradd.md [hunk 9]
```diff
@@ -3,22 +3,30 @@
...
+
+- Create a new user with the home directory filled by template directory files:
+
+`useradd --skel {{path/to/template_directory}} --create-home {{user}}`
+
+- Create a new system user without the home directory:
+
+`useradd --system {{user}}`
```
Comment (navarroaxel) [inline pages/linux/useradd.md:32]:
```suggestion
`useradd --system {{username}}`
```

File: pages/linux/userdel.md [hunk 10]
```diff
@@ -6,16 +6,12 @@
 
 - Remove a user:
 
-`userdel {{name}}`
+`userdel {{user}}`
```
Comment (navarroaxel) [inline pages/linux/userdel.md:9]:
```suggestion
`userdel {{username}}`
```

File: pages/linux/userdel.md [hunk 11]
```diff
@@ -6,16 +6,12 @@
...
 
-- Remove a user along with their home directory and mail spool:
-
-`userdel --remove {{name}}`
-
-- Remove a user from a group:
+- Remove a user in other root directory:
 
-`userdel {{name}} {{group}}`
+`userdel --root {{path/to/other/root}} {{user}}`
```
Comment (navarroaxel) [inline pages/linux/userdel.md:13]:
```suggestion
`userdel --root {{path/to/other/root}} {{username}}`
```

File: pages/linux/userdel.md [hunk 12]
```diff
@@ -6,16 +6,12 @@
...
 
-- Remove a user in other root directory:
+- Remove a user along with the home directory and mail spool:
 
-`userdel --root {{path/to/other/root}} {{name}}`
+`userdel --remove {{user}}`
```
Comment (navarroaxel) [inline pages/linux/userdel.md:17]: g
```suggestion
`userdel --remove {{username}}`
```

File: pages/linux/usermod.md [hunk 13]
```diff
@@ -3,14 +3,22 @@
 > Modifies a user account.
 > More information: <https://manned.org/usermod>.
 
-- Change a user's name:
+- Change a user name:
 
-`usermod -l {{newname}} {{user}}`
+`usermod --login {{new_user}} {{user}}`
```
Comment (navarroaxel) [inline pages/linux/usermod.md:8]:
```suggestion
`usermod --login {{new_username}} {{username}}`
```

File: pages/linux/usermod.md [hunk 14]
```diff
@@ -3,14 +3,22 @@
 > Modifies a user account.
 > More information: <https://manned.org/usermod>.
 
-- Change a user's name:
+- Change a user name:
```
Comment (navarroaxel) [inline pages/linux/usermod.md:6]:
```suggestion
- Change a username:
```

File: pages/linux/usermod.md [hunk 15]
```diff
@@ -3,14 +3,22 @@
...
 
-`usermod -l {{newname}} {{user}}`
+`usermod --login {{new_user}} {{user}}`
 
-- Add user to supplementary groups (mind the whitespace):
+- Change a user id:
 
-`usermod -a -G {{group1,group2}} {{user}}`
+`usermod --uid {{id}} {{user}}`
```
Comment (navarroaxel) [inline pages/linux/usermod.md:12]:
```suggestion
`usermod --uid {{id}} {{username}}`
```

File: pages/linux/usermod.md [hunk 16]
```diff
@@ -3,14 +3,22 @@
...
 
-- Create a new home directory for a user and move their files to it:
+- Change a user shell:
 
-`usermod -m -d {{path/to/home}} {{user}}`
+`usermod --shell {{path/to/shell}} {{user}}`
```
Comment (navarroaxel) [inline pages/linux/usermod.md:16]:
```suggestion
`usermod --shell {{path/to/shell}} {{username}}`
```

File: pages/linux/usermod.md [hunk 17]
```diff
@@ -3,14 +3,22 @@
...
+
+- Add a user to supplementary groups (mind the lack of whitespace):
+
+`usermod --append --groups {{group_a,group_b}} {{user}}`
```
Comment (navarroaxel) [inline pages/linux/usermod.md:20]:
```suggestion
`usermod --append --groups {{group1,group2}} {{username}}`
```

File: pages/linux/usermod.md [hunk 18]
```diff
@@ -3,14 +3,22 @@
...
+
+- Change a user home directory:
+
+`usermod --move-home --home {{path/to/new_home}} {{user}}`
```
Comment (navarroaxel) [inline pages/linux/usermod.md:24]:
```suggestion
`usermod --move-home --home {{path/to/new_home}} {{username}}`
```

File: pages/common/users.md [hunk 19]
```diff
@@ -1,12 +1,12 @@
 # users
 
 > Display a list of logged in users.
-> More information: <https://www.gnu.org/software/coreutils/users>.
+> More information: <https://manned.org/man/users.1>.
```
Comment (sbrl) [inline pages/common/users.md:4]: This should stay as the gnu.org link, since they are the program author

**Review by sbrl — APPROVED** [review]
Thanks, @EmilySeville7cfg!

**Pull request merged**